src/components/Editor/SearchBar.js

```diff
@@ -144,14 +144,24 @@ const SearchBar = React.createClass({
     }
 
     updateQuery(query);
+
```

Comment:
There are some duplicate code in this `doSearch` and `search` in line 204. I am thinking whether write their functionality in only one function or not